falcon/media/json.py

```diff
@@ -26,8 +26,35 @@ class JSONHandler(BaseHandler):
         library's JSON implementation, since it will be faster in most cases
         as compared to a third-party library.
 
-    Overriding the default JSON implementation is simply a matter of specifying
-    the desired ``dumps`` and ``loads`` functions::
+    If you are using the stdlib's json module, you can override the default
```

Comment:
> If you are using the stdlib's json module, 

I think it's best to remove that par since what follows is valid also for external libraries, meaning that I can use partial also with something like `rapidjson` to specify some parameters.

Reply:
Done!